src/plugins/core.js

```diff
@@ -97,9 +98,35 @@ function Plugin(options = {}) {
 
     const chars = initialChars.insert(startOffset, char)
 
+    let nextTransform = state.transform()
```

Comment:
Can you change this variable to just be called `transform` instead of `nextTransform`?